index.js

```diff
@@ -6,13 +6,15 @@ var balanced = require("balanced-match")
 var colorFn = require("css-color-function")
 var helpers = require("postcss-message-helpers")
 
+var colorFnRegExp = /(^|[^\w\-])color\(/;
+
 /**
  * PostCSS plugin to transform color()
  */
 module.exports = postcss.plugin("postcss-color-function", function() {
   return function(style) {
     style.eachDecl(function transformDecl(decl) {
-      if (!decl.value || decl.value.indexOf("color(") === -1) {
+      if (!decl.value || !colorFnRegExp.test(decl.value)) {
```

Comment:
Those micro benchs are not relevant https://www.youtube.com/watch?v=65-RbBwZQdU

I just tried to isolate each tests in a separate process https://gist.github.com/demiazz/b82fd832327367ceba04#comment-1487790

This seems obvious that a stupid indexOf() will be faster than a regex.